src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Linux.cs

```diff
@@ -133,10 +132,14 @@ internal static ProcessInfo CreateProcessInfo(ref Interop.procfs.ParsedStat proc
                 ProcessId = pid,
                 ProcessName = processName ?? Process.GetUntruncatedProcessName(ref procFsStat) ?? string.Empty,
                 BasePriority = (int)procFsStat.nice,
-                VirtualBytes = (long)procFsStat.vsize,
-                WorkingSet = procFsStat.rss * Environment.SystemPageSize,
                 SessionId = procFsStat.session,
-
+                PoolPagedBytes = (long)procFsStatus.VmSwap,
+                VirtualBytes = (long)procFsStatus.VmSize,
+                VirtualBytesPeak = (long)procFsStatus.VmPeak,
+                WorkingSetPeak = (long)procFsStatus.VmHWM,
+                WorkingSet = (long)procFsStatus.VmRSS,
+                PageFileBytes = (long)procFsStatus.VmSwap,
+                PrivateBytes = (long)procFsStatus.VmData,
```

Comment:
I wonder if this should contain the VMStk too. The stack is also private to the process.

Reply:
Yes Indeed, the stack area also needs to be there. Thanks. I will update it to add the stack area.

Reply:
Done.